examples/examples_nodejs_test.go

```diff
@@ -543,9 +545,9 @@ func TestRegress2555(t *testing.T) {
 func TestRegress3421(t *testing.T) {
 	test := getJSBaseOptions(t).
 		With(integration.ProgramTestOptions{
-				Dir: "regress-3421",
-				SkipRefresh: true,
-			},
+			Dir:         "regress-3421",
+			SkipRefresh: true,
```

Comment:
We're missing something in CI to ensure checked in files are gofmt'd seems like.

Reply:
Should we update this to run on PRs? https://github.com/pulumi/pulumi-aws/blob/master/.github/workflows/lint.yml

Reply:
That'd be great! Tickets of this nature can go in https://github.com/pulumi/ci-mgmt